apis/python/src/tiledb/vector_search/index.py

```diff
@@ -490,7 +492,8 @@ def clear_history(
                 if ingestion_timestamp > timestamp:
                     new_ingestion_timestamps.append(ingestion_timestamp)
                     new_base_sizes.append(base_sizes[i])
-                    new_partition_history.append(partition_history[i])
+                    if not is_type_erased_index(index_type):
```

Comment:
Lets add some comment here to explain why we skip for `type_erased_index`

Reply:
Sure, done.